WPMedia/fusion-news-theme-blocks
# fixed styling for results list on mobile view

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

_Information about what you changed for this PR_

## Jira Ticket

- [THEMES-1627](https://arcpublishing.atlassian.net/browse/THEMES-1627-mobile)

## Acceptance Criteria

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout THEMES-1627-mobile`
2. Run fusion repo with linked blocks npx fusion start -f -l @wpmedia/results-list-block
3. Open a desktop page with a results list block
4. Click through different combinations of checkboxes in the block configuration "show promo elements" tab
5. Verify that the description displays correctly with all combinations of promo elements
6. Open a mobile page with a results list block
7. Click through different combinations of checkboxes in the block configuration "show promo elements" tab
8. Verify that the description displays correctly with all combinations of promo elements

## Effect Of Changes

### Before

_Example: When I clicked the search button, the button turned red._

[include screenshot or gif or link to video, storybook would be awesome]

### After

_Example: When I clicked the search button, the button turned green._

[include screenshot or gif or link to video, storybook would be awesome]

## Dependencies or Side Effects

_Examples of dependencies or side effects are:_

- Manifest pr for new block:
- Feature pack pr for local development:
- How to deploy to news: https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3138682964/News+Theme+2.0+Migration+Development+Deployment+Notes
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Dependency on another PR that needs to be merged first

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1627]: https://arcpublishing.atlassian.net/browse/THEMES-1627?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
